source/IAccessibleHandler/orderedWinEventLimiter.py

```diff
@@ -88,11 +90,12 @@ def flushEvents(self):
 		self._genericEventCache = {}
 		threadCounters = {}
 		for k, v in sorted(g.items(), key=lambda item: item[1], reverse=True):
-			threadCount = threadCounters.get(k[-1], 0)
-			if threadCount > MAX_WINEVENTS_PER_THREAD:
-				continue
+			if not alwaysAllowedObjects or k[1:-1] not in alwaysAllowedObjects:
+				threadCount = threadCounters.get(k[-1], 0)
+				threadCounters[k[-1]] = threadCount + 1

```

Comment:
A suggestion: May be an allowed object, while events for it should be allowed, yet should increase the counter. This ensures that focus event floading will block other events.